src/o365/spo/commands/orgassetslibrary/orgassetslibrary-remove.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  ContextInfo, ClientSvcResponse, ClientSvcResponseContents
+} from '../../spo';
+import config from '../../../../config';
+import request from '../../../../request';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandValidate,
+  CommandOption,
+  CommandError
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  confirm?: boolean;
+  libraryUrl: string;
+}
+
+class SpoOrgAssetLibRemoveCommand extends SpoCommand {
```

Comment:
Renamed to `SpoOrgAssetsLibraryRemoveCommand` to match precisely the command `spo orgassetslibrary remove`